### PR TITLE
Doi 710 migrate origin lerna monorepo to rush

### PR DESCRIPTION
**Pull request by royki**
This PR adds rush as monorepo management tool, next to lerna which is still used for NPM releases.

### CODE REVIEW

File: packages/tools/migrations-irec/deployment/migrations/canary/Dockerfile.local [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM energyweb/origin-backend-irec-app:canary
+
+RUN mkdir /var/migration-config
+COPY config/* /var/migration-config/
+
+WORKDIR /usr/local/share/.config/yarn/global/node_modules/\@energyweb/
+
+CMD ["/bin/bash","-c", "packages/apps/origin-backend-irec-app/deployment/packages/apps/origin-backend-irec-app/node_modules/.bin/typeorm migration:run --config exchange/dist/js/ormconfig.js && \
```
Comment (kosecki123) [inline packages/tools/migrations-irec/deployment/migrations/canary/Dockerfile.local]: Is this path correct? I don't think we install anything to `/usr/local/share/.config/yarn/global/node_modules/\@energyweb/`

Comment (royki) [reply packages/tools/migrations-irec/deployment/migrations/canary/Dockerfile.local]: Would it be - 
for
- **migrations** - `WORKDIR /var/deployment/apps/origin-backend-app`
- **migrations-irec** - `WORKDIR /var/deployment/apps/origin-backend-irec-app`

Comment (royki) [reply packages/tools/migrations-irec/deployment/migrations/canary/Dockerfile.local]: @kosecki123 I committed the changes. Can you please check and resolve the conflict.

**Comment by josipbagaric** [comment]
@royki @kosecki123 I'll remove the "Build containers" action as a part of the tests once the PR is approved. It currently makes the build 11 min longer.

**Comment by kosecki123** [comment]
> @royki @kosecki123 Do we want to keep the "Build containers" action as a part of the tests?
> It makes the build 11 min longer.

No. please remove. Also, please keep original Dockerfiles, these based on the NPM installation. The idea is to enable nightly canary releases and use those Dockerfiles to deploy from packages.

**Comment by josipbagaric** [comment]
> > @royki @kosecki123 Do we want to keep the "Build containers" action as a part of the tests?
> > It makes the build 11 min longer.
> 
> No. please remove. Also, please keep original Dockerfiles, these based on the NPM installation. The idea is to enable nightly canary releases and use those Dockerfiles to deploy from packages.

Alright, will do. I didn't remove any Dockerfiles, just added the `Dockerfile.local` ones.

**Comment by kosecki123** [comment]
@JosephBagaric Please squash and merge instead of typical merge, too many useless commit messages.

**Review by kosecki123 — APPROVED** [review]

File: packages/apps/origin-backend-app/Dockerfile.local [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+RUN apk --no-cache add make gcc g++ python bash
+RUN mkdir -p /var/db
+
+RUN mkdir /var/deployment
+COPY ./ /var/deployment
+
+WORKDIR /var/deployment/apps/origin-backend-app
+
+RUN for i in 1 2 3 4 5; do yarn global add @energyweb/origin-backend-app@${VERSION} && break || sleep 15; done
```
Comment (kosecki123) [inline packages/apps/origin-backend-app/Dockerfile.local]: This should not do any NPM installs.

File: packages/apps/origin-backend-irec-app/Dockerfile.local [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+RUN apk --no-cache add make gcc g++ python bash
+RUN mkdir -p /var/db
+
+RUN mkdir /var/deployment
+COPY ./ /var/deployment
+
+WORKDIR /var/deployment/apps/origin-backend-irec-app
+
+RUN for i in 1 2 3 4 5; do yarn global add @energyweb/origin-backend-irec-app@${VERSION} && break || sleep 15; done
```
Comment (kosecki123) [inline packages/apps/origin-backend-irec-app/Dockerfile.local]: This should not do any NPM installs.

File: packages/tools/solar-simulator/Dockerfile.local [hunk 4]
```diff
@@ -0,0 +1,13 @@
+FROM node:14-alpine
+
+ARG VERSION=latest
+
+RUN apk --no-cache add make gcc g++ python
+RUN for i in 1 2 3 4 5; do yarn global add @energyweb/solar-simulator@${VERSION} && break || sleep 15; done
```
Comment (kosecki123) [inline packages/tools/solar-simulator/Dockerfile.local]: This should not do any NPM installs.

**Review by kosecki123 — CHANGES_REQUESTED** [review]
Dockerfiles should not use NPM installs.

**Comment by josipbagaric** [comment]
@kosecki123 is the goal to remove Lerna entirely from the monorepo?
This PR still uses some Lerna commands. I'm thinking of removing them

**Review by kosecki123 — APPROVED** [review]

**Review by josipbagaric — APPROVED** [review]

**Pull request merged**